closure/typedefs/Artifacts.js

```diff
@@ -70,3 +70,6 @@ Artifacts.prototype.screenshots;
 
 /** @type {!Object<!Object>} */
 Artifacts.prototype.criticalRequestChains;
+
+/** @type {!Array<!Object>} */
+Artifacts.prototype.userTimings;
```

Comment:
Seems worthy of an @record extern. I assume we would want to refer to this type elsewhere.